tools/DevicesApiTester/Commands/Pwm/PwmDriverType.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Pwm.Drivers;
+using DeviceApiTester.Infrastructure;
+
+namespace DeviceApiTester.Commands.Pwm
+{
+    public enum PwmDriverType
+    {
+        [ImplementationType(typeof(Windows10PwmDriver))]
+        Windows,
+
+        [ImplementationType(typeof(UnixPwmDriver))]

```

Comment:
I would add this first:
```
        [ImplementationType(null)]
        Default,
```

Then in `PwmCommand.cs`, you would check for `null` specifically in `CreatePwmController()`:
```
            PwmDriver pwmDriver = DriverFactory.CreateFromEnum<PwmDriver, PwmDriverType>(this.Driver);

            return pwmDriver != null
                ? new PwmController(pwmDriver)
                : new PwmController();
```

Reply:
addressed in commit 3